src/main/java/org/blockjam/core/BlockJamCorePlugin.java

```diff
@@ -73,6 +77,10 @@ public static ConfigManager config() {
         return instance().configManager;
     }
 
+    public static BungeeManager bungeeManager() {
+        return instance().bungeeManager;
```

Comment:
This is ugly as hell.

Why does this plugin need to have a static access point? You can retrieve the instance via Sponge's PluginManager. If the plugin isn't loaded, Sponge will give appropriate error messages for the attempt rather than the retrieving plugin getting duped with NPEs.